import express, {Request, Response, NextFunction} from 'express';
import { CatchErrorFunc } from '../utility/CatchErrorFunc';

export const userSignp = CatchErrorFunc(async (req:Request, res:Response) => {

});

export const userLogin = CatchErrorFunc(async (req:Request, res:Response) => {

});

export const verifyUser = CatchErrorFunc(async (req:Request, res:Response) => {

});

export const requestUserOTP = CatchErrorFunc(async (req:Request, res:Response) => {

});

export const getUserProfile = CatchErrorFunc(async (req:Request, res:Response) => {

});

export const updateUserProfile = CatchErrorFunc(async (req:Request, res:Response) => {

});